### PR TITLE
User can't login with blank "Username" textbox

### PR DESCRIPTION
**Pull request by Baonganbui123**

### CODE REVIEW

File: src/test/java/tests/LoginTests.java [hunk 1]
```diff
@@ -1,19 +1,39 @@
...
+    @Test(description = "User can't login with blank 'Username' textbox")
+    public void tc002_LoginWithInValidInfo() {
+
+        LogHelpers.printInfo("Click on 'Login' tab");
+        loginPage.clickLoginTab();
+
+        LogHelpers.printInfo("User doesn't type any words into 'Username' textbox but enter valid information into 'Password' textbox ");
+        loginPage.login("", Constants.VALID_PASSWORD);
+
+        LogHelpers.printInfo("Show error message");
```
Comment (tanmle) [inline src/test/java/tests/LoginTests.java:36]: it should say 'Verify that.....`

File: src/test/java/tests/LoginTests.java [hunk 2]
```diff
@@ -1,19 +1,39 @@
...
     @Test(description = "User can log into Railway with valid username and password")
     public void tc001_LoginWithValidUsernameAndPassword() {
+
+        LogHelpers.printInfo("Click on 'Login' tab");
         loginPage.clickLoginTab();
+
+        LogHelpers.printInfo("Enter valid username and password");
         loginPage.login(Constants.VALID_USERNAME, Constants.VALID_PASSWORD);
 
+        LogHelpers.printInfo("Show welcome message");
```
Comment (tanmle) [inline src/test/java/tests/LoginTests.java:23]: it should say 'Verify that.....`

File: src/test/java/tests/LoginTests.java (hunk 1 above)
Comment (Baonganbui123) [reply src/test/java/tests/LoginTests.java]: Changed

File: src/test/java/tests/LoginTests.java (hunk 2 above)
Comment (Baonganbui123) [reply src/test/java/tests/LoginTests.java]: Changed

**Review by tanmle — APPROVED** [review]

**Review by anhlvt — APPROVED** [review]

**Pull request merged**